llms/mlx_lm/server.py

```diff
@@ -474,15 +531,15 @@ def handle_completion(
 
     def handle_stream(
         self,
-        prompt: mx.array,
+        prompt: List[int],
```

Comment:
This in particular.  Can't we handle a single or multiple (batched) prompt that falls back to the behavior for a single prompt by default?

Reply:
We can always update this type to `List[List[int]]` when the time comes.